plugins/module_utils/foreman_helper.py

```diff
@@ -92,10 +92,12 @@ def inner(self, *args, **kwargs):
     return decor
 
 
-def _check_patch_needed(fixed_version=None, plugins=None):
+def _check_patch_needed(introduced_version=None, fixed_version=None, plugins=None):
     """
     Decorator to check whether a specific apidoc patch is required.
 
+    :param introduced_version: The version of Foreman the API bug was introduced.
+    :type introduced_version: str, optional
```

Comment:
Somehow i was missing this, but didn't want to ask...

Reply:
My very first version of that had it, but then I was like "fuck, we'll never need it". Well, never didn't last a week.